app/views/report/_export_widgets.haml

```diff
@@ -73,7 +73,7 @@
 
     setTimeout(function() {
       if (typeof(dhxLayoutB) !== 'undefined') {
-        dhxLayoutB.cells('c').collapse();
+        ManageIQ.layout.toolbar.hide();
         $(dhxLayoutB.cells('c')).height(1);
```

Comment:
@skateman Unfortunately, I do not know. I went back and looked at the commit thinking that I must've moved it from somewhere but I can't find anywhere that indicates that it was moved, so I'm not sure what this is for. It definitely doesn't ring any bells. Sorry :cry: